SConstruct

```diff
@@ -79,7 +86,13 @@ def set_defaults(env):
                         '-fpic', '-D_GNU_SOURCE', '-DD_LOG_V2'])
     env.Append(CCFLAGS=['-O2', '-DDAOS_VERSION=\\"' + DAOS_VERSION + '\\"'])
     env.Append(CCFLAGS=['-DCMOCKA_FILTER_SUPPORTED=0'])
+    env.Append(CCFLAGS=['-D_FORTIFY_SOURCE=2'])
```

Comment:
This (and the other -D options) should all be using CPPDEFINES rather than CCFLAFGS really although this isn't a regression.